lib/google/storage/property/defaultobjectacl_project_team.rb

```diff
@@ -94,9 +93,7 @@ def initialize(args)
       # Data is coming from the Puppet manifest
       class DefaObjeAclProjTeamCatalog < DefaObjeAclProjTeam
```

Comment:
Can we make these names saner sometime soon?

Reply:
Yes, definitely!